### ISSUE TITLE
Not possible to search words on task list headers

### ISSUE THREAD

**Issue by nickelaos** [streetwriters/notesnook#4308]
### What happened?

I have a lot of notes with task lists (lists with checkboxes). When I try to search some content within the note, I find only list items. But I also want to search through list titles!

### Steps to reproduce the problem

1. Create a note with a few task lists
2. Press Ctrl+F and search for the word from the task header

![image](https://github.com/streetwriters/notesnook/assets/21063721/1afbfff4-9562-4bb3-9991-5a6660a1656a)

### Version

2.6.15

### Platform/OS

Windows, Chrome/Chromium

### Relevant log output

*No response*

**Comment by thecodrr** [comment]
Yeah that's going to be difficult. The task list header is not part of the editor. It appears inside the editor but the normal search doesn't include it while searching. I'll have to look deeper and see if this is possible.
